solidity-v1/dashboard/src/pages/threshold/index.js

```diff
@@ -0,0 +1,17 @@
+import React from "react"
+import HowItWorksPage from "./HowItWorksPage"
+import ThresholdUpgradePage from "./ThresholdUpgradePage"
+import PageWrapper from "../../components/PageWrapper"
+
+const ThresholdPageContainer = ({ title, routes, withNewLabel }) => {
+  return <PageWrapper title={title} routes={routes} newPage={withNewLabel} />
+}
+
+ThresholdPageContainer.route = {
+  title: "Threshold",
```

Comment:
Based on the Figma views it should be `Threshold Upgrade`.

Reply:
Fixed in d298cb63a2ae9f64da792e0d13c963b4666af797